components/home/Home.jsx

```diff
@@ -1,85 +1,68 @@
 import * as React from 'react';
-import { View, StyleSheet } from 'react-native';
-import { Button, Title } from 'react-native-paper';
-import WorkoutContainer from './WorkoutContainer';
+import {
+  SafeAreaView, Image, View,
+} from 'react-native';
+import styled from 'styled-components/native';
+import CurrentCycle from './CurrentCycle';
+import WorkoutList from './WorkoutList';
 
-export default () => (
-  <View style={styles.container}>
+const Title = styled.Text`
+  font-family: 'Montserrat_700Bold';
+  font-size: 40px;
+  margin: 0px 6%;
+`;
 
-    {/* Cycles Section */}
-    <View style={{ ...styles.sectionComponent, margin: 30 }}>
+const welcomeName = 'Justin';
 
-      <Title style={styles.title}>
-        Cycles
-      </Title>
-
-      <Button contentStyle={{ width: 331, height: 170 }} style={styles.startCycle} color="#FFFFFF" onPress={() => alert('This should start the current workout in the cycle')}>
-        Start Cycle Workout
-      </Button>
-
-      <Button contentStyle={{ width: 325, height: 35 }} style={styles.seeAllButton} color="#000000" onPress={() => alert('This should take you to workouts tab')}>
-        See All Cycles
-      </Button>
-
-    </View>
-
-    {/* Workouts Section */}
-    <View style={{ ...styles.sectionComponent, margin: 10 }}>
-      <Title style={styles.title}>
-        Workouts
-      </Title>
-
-      <WorkoutContainer
-        items={[{ name: 'Push Workout A' }, { name: 'Pull Workout A' }]}
+export default () => {
+  const items = [
+    {
+      name: 'Lateral Pulldo ...', subtext: '1 day ago', color: '#CAB0FF', onPress: () => alert('Push, Pull, Legs A'),
```

Comment:
Make sure that you cap the amount of letters shown.

Reply:
done